test/tbb/test_parallel_phase.cpp

```diff
@@ -45,7 +45,8 @@ struct dummy_func {
 template <typename F1 = dummy_func, typename F2 = dummy_func>
 std::size_t measure_median_start_time(tbb::task_arena* ta, const F1& start = F1{}, const F2& end = F2{}) {
     std::size_t num_threads = ta ? ta->max_concurrency() : tbb::this_task_arena::max_concurrency();
-    std::size_t num_runs = 500;
+    std::size_t max_wait = 1000;
+    std::size_t num_runs = 100;
```

Comment:
Will something like the following not lose much on the testing quality and its stability while reducing the testing time even more?
```cpp
const std::size_t waitings = [0, 10, 50, 100, 500, 1000, 2000]`;
for (auto const& i : waitings) {
   if (ta) {
       ta->execute(work);
   } else {
       work();
   }
   active_wait_for(std::chrono::microseconds(i));
}
```

Reply:
1. I'm leaning towards having more steps so test would output more accurate data on worker threads' new work start time.
2. 2000 waiting time won't make much sense here because it will exceed worker threads block time, meaning there will be no difference in start times between `fast_leave` and `automatic` policies.

Reply:
1. Okay.
2. Does it make sense to propagate the worker threads block time to the test as `max_wait` then? This will automatically cover the cases if the block time changes.

Reply:
Yes, it makes sense. Added the corresponding TODO.